### ISSUE TITLE
Add Bundler benchmark

### ISSUE THREAD

**Issue by noahgibbs** [Shopify/yjit-bench#113]
Alan points out that a "run Bundler with a big Gemfile but with all relevant gems already installed" benchmark would address a fair number of real-world compile time worries. If our compile time, or similar startup time, gets too slow then this benchmark would clearly show the real-world impact.

**Comment by maximecb** [comment]
Yes, but I would also like to see a synthetic benchmark that more directly tests compile time, like the 30k_ifelse benchmark, but with a smaller number of iterations (just enough to get everything to compile).

**Comment by noahgibbs** [comment]
Absolutely. That's this issue: https://github.com/Shopify/yjit-bench/issues/114
